test/Atmos/EDMF/closures/mixing_length.jl

```diff
@@ -91,9 +91,9 @@ function mixing_length(
     w_up = vuntuple(i -> up[i].ρaw / up[i].ρa, N_up)
     b = sum(
         ntuple(N_up) do i
-            a_up[i] * w_up[i] * δ[i] / env.a *
+            a_up[i] * w_up[i] * Δ[i]/max(up[i].ρaw,eps(FT)) / env.a *
```

Comment:
we need to decide how to better handle the case of Δ=ρawδ=0 when w=0. (w=0 is an allowed value, in contrast to a=0 which should be circumvented elsewhere in the code).

Reply:
In the entr_detr function, we could return both E_dyn, Δ_dyn, E_trb and ε_dyn, δ_dyn, ε_trb (or a subset of that). That way, we can deal with 0/0 problems at the source of the problem, and pass the fractional rates here.